### DISCUSSION TITLE
Maintenance question: Should we start opening PRs against react-editor-view?

### DISCUSSION THREAD

**Discussion by smoores-dev** [nytimes/react-prosemirror#87] [General]
Some folks have started giving the new architecture a shot and opening issues about it. Do we feel like the react-editor-view branch is stable enough that we should start opening PRs against it?

Part of this question is: do folks other than me feel comfortable enough with the react-editor-view branch to review PRs opened against it?

I will probably start doing this for now, but I'd like to hear what folks think!

Tagging @coolov @tilgovi and @eelzon, would love input from anyone, though!

**Comment by coolov** [comment]
i don't yet feel comfortable enough with the branch to review prs, but i don't mind looking at them as i'm sure this will teach me a bunch. feel free to do what you think is best (as long as you don't make me a blocking reviewer :))

**Reply by smoores-dev** [reply]
Ok! That sounds good to me. There are currently no restrictions for merging PRs into that branch, so I'll open PRs to give folks a chance to see the changes and ask questions, but I'll probably just merge them after a day or so for expediency's sake
